components/brave_wallet/browser/android_page_appearing_browsertest.cc

```diff
@@ -425,14 +425,16 @@ IN_PROC_BROWSER_TEST_F(AndroidPageAppearingBrowserTest,
                        TestDepositPageAppearing) {
   GURL url = GURL("chrome://wallet/deposit-funds");
   const std::vector<std::string> ignore_patterns = {
-      "TypeError: Cannot read properties of undefined (reading 'forEach')"};
+      "TypeError: Cannot read properties of undefined (reading 'forEach')",
+      "TypeError: Cannot read properties of undefined (reading 'match')"};
```

Comment:
Adding this line breaks the test, as it now passes but the deposit page does not load correctly.

<img width="647" alt="Screenshot 2023-08-03 at 4 22 16 PM" src="https://github.com/brave/brave-core/assets/3308503/836cd629-7e0f-4f30-9302-91325377dade">